### PR TITLE
[1LP][RFR] RHV tiers for several tests

### PR DESCRIPTION
**Pull request by jan-zmeskal**
RHV tier markers for several test cases

### CODE REVIEW

File: cfme/tests/infrastructure/test_snapshot.py [hunk 1]
```diff
@@ -243,6 +243,7 @@ def test_revert_active_snapshot(full_test_vm, provider, soft_assert, register_ev
                            active_snapshot=True)
 
 
+@pytest.mark.rhv3
```
Comment (istein1) [inline cfme/tests/infrastructure/test_snapshot.py]: I think this should be tier 2

File: cfme/tests/infrastructure/test_vm_clone.py [hunk 2]
```diff
@@ -57,6 +57,7 @@ def test_vm_clone(appliance, provider, clone_vm_name, create_vm):
     assert request_row.is_succeeded(method='ui'), msg
 
 
+@pytest.mark.rhv3
```
Comment (istein1) [inline cfme/tests/infrastructure/test_vm_clone.py]: The functionality of clone VM (create a VM from existing one) exist on RHV, but do not exist on CFME.
Not sure why this test is run for RHV provider.

File: cfme/tests/infrastructure/test_snapshot.py (hunk 1 above)
Comment (jan-zmeskal) [reply cfme/tests/infrastructure/test_snapshot.py]: @istein1 Actually here I have to disagree. If you check the description, it just checks one button for being clickable. I cannot imagine such test anywhere out of tier3.

File: cfme/tests/infrastructure/test_vm_clone.py (hunk 2 above)
Comment (istein1) [reply cfme/tests/infrastructure/test_vm_clone.py]: Great, thanks for explaining

File: cfme/tests/infrastructure/test_snapshot.py (hunk 1 above)
Comment (istein1) [reply cfme/tests/infrastructure/test_snapshot.py]: After discussion on what this test do offline, I agree it should be tier 3.

**Pull request merged**